introduction/java_if_else/java_if_else.java

```diff
@@ -0,0 +1,26 @@
+import java.util.Scanner;
+
+public class Solution {
+  
+  private static final String WEIRD = "Weird";
+  private static final String NOT_WEIRD = "Not Weird";
+  
+  private static String isWeird(int numberToCheck) {
```

Comment:
return bool and convert to string later. I think that was already mentioned in previous comments.